bootstrap/bootstrap.go

```diff
@@ -219,6 +220,30 @@ func startServer(
 		signer, err = crypto.NewInMemorySigner(cfg.COAKey, crypto.SHA3_256)
 	case cfg.COAKeys != nil:
 		signer, err = requester.NewKeyRotationSigner(cfg.COAKeys, crypto.SHA3_256)
+	case cfg.COACloudKMSKeys != nil:
+		kmsKeys := make([]flowGoKMS.Key, len(cfg.COACloudKMSKeys))
+		for i, keyID := range cfg.COACloudKMSKeys {
+			kmsKeys[i] = flowGoKMS.Key{
+				ProjectID:  cfg.COACloudKMSProjectID,
+				LocationID: cfg.COACloudKMSLocationID,
+				KeyRingID:  cfg.COACloudKMSKeyRingID,
+				KeyID:      keyID,
+				KeyVersion: "1",
```

Comment:
@sideninja Any suggestions on how to manage key versions? Do we want to support this from day 1?

Reply:
I have added a basic support for this in: https://github.com/onflow/flow-evm-gateway/pull/282/commits/2d7a22c054ef463ba2dca7fbf7bd0950f6032be0